pages/linux/pacman.md

```diff
@@ -5,32 +5,32 @@
 
 - Synchronize and update all packages:
 
-`pacman -Syu`
+`pacman --sync --refresh --sysupgrade`
 
 - Install a new package:
 
-`pacman -S {{package_name}}`
+`pacman --sync {{package_name}}`
 
 - Remove a package and its dependencies:
 
-`pacman -Rs {{package_name}}`
+`pacman --remove --recursive {{package_name}}`
 
 - Search the package database for a regular expression or keyword:
 
-`pacman -Ss "{{search_pattern}}"`
+`pacman --sync --search "{{search_pattern}}"`
 
 - List installed packages and versions:
 
-`pacman -Q`
+`pacman --query`
 
 - List only the explicitly installed packages and versions:
 
-`pacman -Qe`
+`pacman --query --explicit`
 
-- Find which package owns a certain file:
+- List orphan packages (installed as dependencies but not required by any package):
 
-`pacman -Qo {{filename}}`
+`pacman --query --unrequired --deps --quiet`
 
-- Empty package cache to free up space:
+- Remove not installed packages and unused repositories from the cache (use two `--clean` flags to clean all packages):
```

Comment:
I don't think this is completely correct. `-c` removes all uninstalled packges and `-cc` removes all files from the cache, regardless if they are a package or something else. So I think the original description is fine.

Reply:
What about
```suggestion
Delete the entire cache to free up space
```

Reply:
Empty implies that some junk is removed. Delete is more extreme which is what clean clean does

Reply:
I think both `Empty` and `Delete` are fine here. Personally I'm inclined to agree with @marchersimon on this one.

Also for reference the `pacman` command has been the subject of frequent changes in the past: https://github.com/tldr-pages/tldr/commits/main/pages/linux/pacman.md

Reply:
@marchersimon you don't like the current description? It's the same that the last example in the `pacman-sync` page.

Reply:
`Remove all not installed packeges` describes what `-Sc` does. `-Scc` however empties the entire cache, regardless if the file belongs to a non-installed package.

Reply:
The last example of `pacman-sync` is correct, because it does describe `-Sc`.

Reply:
```suggestion
- Delete the entire pacman cache:
```

Reply:
I'd preferr `empty the entire pacman cache`, since I think it's used more often in context with cache. "Delete the cache" sounds a little off IMO.

Reply:
```suggestion
- Empty the entire pacman cache:
```